### PR TITLE
Add new cmdlets

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@get-itips @dariomws @tseward

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -2136,6 +2141,99 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -CoreSharingCapability
+
+Determines what level of sharing is available for the SharePoint Sites (Does not include OneDrive sites)
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2146]:
```suggestion
Determines what level of sharing is available for SharePoint sites (not including OneDrive sites).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -2136,6 +2141,99 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: ExternalUserAndGuestSharing
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -OneDriveRequestFilesLinkEnabled
+
+Enable or Disable request files link on OneDrive partition for all OneDrive sites. If this value is not set, Request Files will only show for OneDrives with Anyone Links enabled.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2171]:
```suggestion
Enable or disable the Request files link on the OneDrive partition for all OneDrive sites. If this value is not set, the Request files link will only show for OneDrives with Anyone links enabled. 
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 3]
```diff
@@ -2136,6 +2141,99 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CoreRequestFilesLinkEnabled
+
+Enable or Disable request files link on OneDrive partition for all SharePoint sites (Not OneDrive Sites). If this value is not set, Request Files will only show for OneDrives with Anyone Links enabled.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2187]:
```suggestion
Enable or disable the Request files link on the OneDrive partition for all SharePoint sites (not including OneDrive sites). If this value is not set, Request files will only show for OneDrives with Anyone links enabled. 
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 4]
```diff
@@ -2136,6 +2141,99 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -OneDriveRequestFilesLinkExpirationInDays
+
+Specifies the number of days before an Request files link expires for all OneDrive Sites.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2203]:
```suggestion
Specifies the number of days before a Request files link expires for all OneDrive sites. 
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 5]
```diff
@@ -2136,6 +2141,99 @@ Accept pipeline input: False
...
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -OneDriveRequestFilesLinkExpirationInDays
+
+Specifies the number of days before an Request files link expires for all OneDrive Sites. 
+
+Value can be from 0 to 730 days.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2205]:
```suggestion
The value can be from 0 to 730 days.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 6]
```diff
@@ -2136,6 +2141,99 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: -1
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CoreRequestFilesLinkExpirationInDays
+
+Specifies the number of days before an Request files link expires for all SharePoint sites (Not OneDrive sites)
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2221]:
```suggestion
Specifies the number of days before a Request files link expires for all SharePoint sites (not including OneDrive sites).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 7]
```diff
@@ -2136,6 +2141,99 @@ Accept pipeline input: False
...
+
+Value can be from 0 to 730 days.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2223]:
```suggestion
The value can be from 0 to 730 days.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @salarson, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**